lib/stringex/localization/converter.rb

```diff
@@ -59,7 +59,11 @@ def translate!(*conversions)
         end
       end
 
-    protected
+      protected
+
+      def unreadable_control_characters
+        string.gsub! expressions.unreadable_control_characters, ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.